pydis_site/apps/api/viewsets/bot/user.py

```diff
@@ -16,6 +18,30 @@ class UserListPagination(PageNumberPagination):
     page_size = 10000
     page_size_query_param = "page_size"
 
+    def get_next_page_number(self) -> int:
+        """Get the next page number."""
+        if not self.page.has_next():
+            return None
+        page_number = self.page.next_page_number()
+        return page_number
+
+    def get_previous_page_number(self) -> int:
+        """Get the previous page number."""
+        if not self.page.has_previous():
+            return None
+
+        page_number = self.page.previous_page_number()
+        return page_number
+
+    def get_paginated_response(self, data: list) -> Response:
+        """Override method to send modified response."""
+        return Response(OrderedDict([
+            ('count', self.page.paginator.count),
+            ('next_page_no', self.get_next_page_number()),
+            ('previous_page_no', self.get_previous_page_number()),
+            ('results', data)
```

Comment:
Don't forget to update the view docs since you've changed the paginator.

Reply:
right :+1: